idb/grpc/client.py

```diff
@@ -789,6 +791,48 @@ async def list_xctests(self) -> List[InstalledTestInfo]:
     async def send_events(self, events: Iterable[HIDEvent]) -> None:
         await self.hid(iterator_to_async_iterator(events))
 
+    @log_and_handle_exceptions
+    async def axtap(
+        self,
+        element: HIDElementType,
+        label: str,
+        x: float,
+        y: float,
+        duration: Optional[float] = None,
+        count: Optional[int] = 1,
+    ) -> None:
+        accessibilityInfo = await self.stub.accessibility_info(
+            AccessibilityInfoRequest(
+                point=None,
+                format=(AccessibilityInfoRequest.LEGACY),
+            )
+        )
+
+        elementFound = False
+
+        for item in json.loads(accessibilityInfo.json):
+            try:
+                axElement = HIDElementType(item["role_description"])
+                axLabel = item["AXLabel"]
+
+                elementFound = axElement == element and (
```

Comment:
For all the element types except for `back`, does it make sense to allow commands without a `label`?

The current behavior is that:

```
$ idb ui axtap button
```

Succeeds and returns the first `button` it encounters, right?

Reply:
Yes, it just uses the first element of the type.

It's debatable where or not that is useful because I don't know if we can guarantee that the order of `accessibilityInfo.json` is the same every run, though it appeared that way when testing.